source/extensions/filters/http/jwt_authn/matcher.cc

```diff
@@ -188,6 +189,35 @@ class PathSeparatedPrefixMatcherImpl : public BaseMatcherImpl {
 };
 } // namespace
 
+class PathMatchPolicyMatcherImpl : public BaseMatcherImpl {
+public:
+  PathMatchPolicyMatcherImpl(const RequirementRule& rule,
+                             Server::Configuration::CommonFactoryContext& context)
+      : BaseMatcherImpl(rule, context) {
+    auto& factory = Config::Utility::getAndCheckFactory<Router::PathMatcherFactory>(
+        rule.match().path_match_policy());
+    ProtobufTypes::MessagePtr config = Envoy::Config::Utility::translateAnyToFactoryConfig(
+        rule.match().path_match_policy().typed_config(),
+        ProtobufMessage::getStrictValidationVisitor(), factory);
+
+    uri_template_matcher_ = factory.createPathMatcher(*config).value();
```

Comment:
createPathMatcher returns `absl::StatusOr<PathMatcherSharedPtr>`

It will good to check the return status and fail early to prevent nullptr access.

Reply:
hey, I made the changes, can you take a look?